src/server.tsx

```diff
@@ -23,6 +23,14 @@ const NODE_ENV = process.env.NODE_ENV;
 const PROD = NODE_ENV === 'production';
 const DEV = !PROD;
 
+const RESTRICTIVE_ROBOTS_TXT = `User-agent: *
+Disallow: /`;
+
+const OPEN_ROBOTS_TXT = `User-agent: *
+Disallow: /g/chess/online/*
+Disallow: /g/seabattle/online/*
```

Comment:
We have access to the GAMES_LIST constant. Can we generate this based on that constant? So we dont need to hard code values